shared/src/types/types.ts

```diff
@@ -113,6 +113,7 @@ export interface ChatMessage {
   value: { message: string } | Record<string, unknown>;
   chatToken?: string;
   messageToken: string;
+  timestamp: string;
```

Comment:
This annoys me for no apparent reason.  I know it's fine, but geeze Typescript, maybe make it a bit easier to say "ISO-8601 formatted datetime"?